fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -292,17 +366,25 @@ private fun Modifier.bottomDrawerSwipeable(
             val minHeight = 0f
             val bottomOpenStateY = max(maxOpenHeight, fullHeight - drawerHeight)
             val bottomExpandedStateY = max(minHeight, fullHeight - drawerHeight)
-            val anchors = if (drawerHeight <= maxOpenHeight || !expandable) {
+            val anchors = if (drawerHeight <= maxOpenHeight || drawerState.skipOpenState) {
                 mapOf(
                     fullHeight to DrawerValue.Closed,
                     bottomOpenStateY to DrawerValue.Open
                 )
             } else {
-                mapOf(
-                    fullHeight to DrawerValue.Closed,
-                    bottomOpenStateY to DrawerValue.Open,
-                    bottomExpandedStateY to DrawerValue.Expanded
-                )
+                if(drawerState.expandable){
+                    mapOf(
+                        fullHeight to DrawerValue.Closed,
+                        bottomOpenStateY to DrawerValue.Open,
+                        bottomExpandedStateY to DrawerValue.Expanded
+                    )
+                }
+                else {
```

Comment:
else part wont be required as per above comment

Reply:
now this else contains check for skipOpenState and passes the second parameter accordingly.